src/scripts/main.js

```diff
@@ -22,7 +22,31 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  element.innerHTML = createContent(data);
+}
+
+function createContent(obj) {
+  let li = '';
+  let ul;
+
+  for (const key in obj) {
+    li += `
+      <li>
+        ${key}
+        ${createContent(obj[key])}
+      </li>
+    `;
+  }
+
+  if (li) {
+    ul = `
+      <ul>
+        ${li}
+      </ul>
+    `;
```

Comment:
Please, rewrite your solution with dom manipulations. For example to create an element use `document.createElement()`, to add an item to the list use the `append()` method and so on